src/common/node/childProcess.ts

```diff
@@ -55,44 +61,32 @@ export class ChildProcess {
         return this.exec(command, options).outcome.then(stdout => stdout.toString());
     }
 
-    public spawnWithExitHandler(command: string, args?: string[], options: ISpawnOptions = {}): ISpawnResult {
+    public spawn(info: ISpawnInfo, options: ISpawnOptions = {}): ISpawnResult {
```

Comment:
Another way of going would be to merge the `ISpawnInfo` and `ISpawnOptions` interfaces. We only use them in this file, and we always use them together.